<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>725</width>
    <height>830</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="reload_button">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Reload from selection</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="replace_groupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Replace</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="replace_layout">
         <item>
          <widget class="QLineEdit" name="replace_line"/>
         </item>
         <item>
          <widget class="QLabel" name="with_label">
           <property name="text">
            <string>  with : </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="replacewith_line"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="matchcase_checkbox">
         <property name="text">
          <string>Match case</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="remove_groupBox">
      <property name="title">
       <string>Remove</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="firstlast_layout">
         <item>
          <widget class="QLabel" name="fisrt_label">
           <property name="text">
            <string>First :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="first_spin"/>
         </item>
         <item>
          <widget class="QLabel" name="last_label">
           <property name="text">
            <string>Last :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="last_spin"/>
         </item>
         <item>
          <spacer name="firstlast_spacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="fromto_layout">
         <item>
          <widget class="QLabel" name="from_label">
           <property name="text">
            <string>From :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="from_spin"/>
         </item>
         <item>
          <widget class="QLabel" name="to_label">
           <property name="text">
            <string>To :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="to_spin"/>
         </item>
         <item>
          <spacer name="fromto_spacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="add_groupBox">
      <property name="title">
       <string>Add</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <layout class="QHBoxLayout" name="prefix_layout">
         <item>
          <widget class="QLabel" name="prefix_label">
           <property name="text">
            <string>Prefix :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="prefix_line"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="insert_layout">
         <item>
          <widget class="QLabel" name="insert_label">
           <property name="text">
            <string>Insert :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="insert_line"/>
         </item>
         <item>
          <widget class="QLabel" name="insertat_label">
           <property name="text">
            <string>at :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="insertat_spin"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="suffix_layout">
         <item>
          <widget class="QLabel" name="suffix_label">
           <property name="text">
            <string>Suffix :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="suffix_line"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="numbering_groupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Numbering</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="mode_layout">
         <item>
          <widget class="QLabel" name="mode_label">
           <property name="text">
            <string>Mode :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="mode_combo">
           <item>
            <property name="text">
             <string>None</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Suffix</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Prefix</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Insert</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="modeat_label">
           <property name="text">
            <string>  at :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="modeat_spin"/>
         </item>
         <item>
          <spacer name="mode_spacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="startincrement_layout">
         <item>
          <widget class="QLabel" name="start_label">
           <property name="text">
            <string>Start :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="start_spin"/>
         </item>
         <item>
          <widget class="QLabel" name="increment_label">
           <property name="text">
            <string>Increment :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="increment_spin"/>
         </item>
         <item>
          <spacer name="startincrement_spacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="padding_layout">
         <item>
          <widget class="QLabel" name="padding_label">
           <property name="text">
            <string>Padding :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="padding_spin"/>
         </item>
         <item>
          <spacer name="padding_spacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
